lib/individual/health.dart

```diff
@@ -14,169 +15,249 @@ class _HealthPageState extends State<HealthPage> {
-                                    suffixIcon: const Icon(
-                                      Icons.arrow_drop_down,
-                                      color: Colors.grey,
-                                      size: 40,
+                                    // SizedBox(
+                                    //   width: 130.w,
+                                    // ),
+                                    // Text(
+                                    //   "Time",
+                                    //   style: TextStyle(
+                                    //       fontWeight: FontWeight.w500,
+                                    //       fontSize: 20.sp),
+                                    // ),
+                                    SizedBox(
+                                      width: 40.w,
                                     ),
-                                  ),
+                                    // SizedBox(
+                                    //   height: 10.h,
+                                    // )
+                                    //
+                                  ],
                                 ),
-                              ),
-                            ],
-                          )
-                        ]),
-                  ),
-                ],
-                // ]
+                                Row(
+                                  mainAxisAlignment: MainAxisAlignment.center,
+                                  children: [
+                                    Padding(
+                                      padding:
+                                          EdgeInsets.fromLTRB(15.w, 0, 15.w, 0),
+                                    ),
+                                    Expanded(
+                                      child: TextField(
+                                        decoration: InputDecoration(
+                                            fillColor: const Color.fromARGB(
+                                                255, 186, 231, 235),
+                                            filled: true,
+                                            suffixIcon: const Icon(
+                                              Icons.arrow_drop_down,
+                                              color: Colors.grey,
+                                              size: 40,
+                                            ),
+                                            hintText: 'Date',
+                                            border: OutlineInputBorder(
+                                                borderSide: BorderSide(
+                                                    color: Colors.grey,
+                                                    width: 2.w),
+                                                borderRadius:
+                                                    BorderRadius.circular(80))),
+                                      ),
```

Comment:
In the Design file, there is also an option to select **Time**. You may use **TimePicker** to achieve the outcome.
Please add this functionality into the page as well. 

Reference:
> https://stackoverflow.com/questions/63223855/what-is-the-correct-way-to-add-time-picker-in-flutter-app

Reply:
Solved!!! :sparkles: